### PR TITLE
Dom manipulation

### PR DESCRIPTION
**Pull request by audukehinde**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -48,21 +47,137 @@ form.addEventListener('submit', (e) => {
...
+projectSectionTitle.className = 'recent';
+underLineImage.className = 'line-divider2';
+projectCard.className = 'inner-element';
+
+underLineImage.src = 'img/Vector4.png';
+underLineImage.setAttribute('alt', 'Project Image');
+projectSectionTitle.innerText = 'MY RECENT PROJECT';
+projectSection.append(projectSectionTitle, underLineImage);
+
+const projectData = [
```
Comment (SajjadAhmad14) [inline main.js:121]: Pop-up should close when the user clicks on the `X` button.

File: main.js [hunk 2]
```diff
@@ -48,21 +47,137 @@ form.addEventListener('submit', (e) => {
...
+  4,
+  {
```
Comment (SajjadAhmad14) [inline main.js:123]: Please use the same background color for the pop-up container as in the design. Moreover please add icons to the buttons,
design
background
icons

File: main.js [hunk 3]
```diff
@@ -48,21 +47,137 @@ form.addEventListener('submit', (e) => {
...
+    projectTitle: '',
+    projectDescription: "A website created for a digital marketing agency to help improve sales and give information about company's product. This website also helps the client to contact the company for a business deal and showcases the company's different services and packages.",
+  },
+  {
```
Comment (SajjadAhmad14) [inline main.js:127]: Buttons should stack on each other for mobile view. Please follow the design and make the pop-up on mobile the same as in the design
mob
.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Kenny Audu, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏
### Please always use the first Pull Request for all the re-reviews so that reviewer can review your code only for the previously requested changes.
- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: main.js (hunk 1 above)
Comment (audukehinde) [reply main.js:121]: The X button is functional sir. It closes when it is clicked

File: main.js [hunk 4]
```diff
@@ -48,21 +47,138 @@ form.addEventListener('submit', (e) => {
...
+  projectSection.innerHTML += projects;
+}
 
-// // console.log(projectData.projectLanguages.thirdLang);
+// eslint-disable-next-line no-multi-assign
+const popupcard = document.getElementById('card');
+const popcard = `<div class="card-header">
+    <h2>Multi Post Story</h2>
+  <div class="class-close">&times;</div>
+  </div>
```
Comment (safafa) [inline main.js:167]: - [ ] Kindly fix the x button. it should close the popup card when it's clicked.

File: index.html [hunk 5]
```diff
@@ -249,6 +244,26 @@ <h3>CONTACT US</h3>
...
+        <div class="toolx">
+          <ul>
+            <li class="line">CSS</li>
+            <li class="line">html</li>
+            <li class="line">Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button class="project-link btnsee">See Project</button>
+        <button class="project-link btnsee">See Project</button> -->
```
Comment (safafa) [inline index.html:265]: - [ ] [OPTIONAL] Kindly remove any commented code. to increase the code readability.

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi team, Good job working on some of the changes requested by the previous reviewer 👏🏼 👏🏼 

But,  there are still some which aren't addressed yet.

## Required changes :

- [ ] Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 5 above)
Comment (audukehinde) [reply index.html:265]: This is not nice. The X button is working. This was my last review and you have successfully made me repeat the week.

Comment (audukehinde) [reply index.html:265]: Check the project very well next time

**Pull request closed without merging**